spyder_kernels/console/kernel.py

```diff
@@ -972,16 +946,54 @@ def _set_config_option(self, option, value):
         except Exception:
             pass
 
-    def _set_mpl_inline_rc_config(self, option, value):
+    def _set_inline_config_option(self, option, value):
         """
-        Update any of the Matplolib rcParams given an option and value.
+        Update InlineBackend given an option and value.
+
+        As parameters:
+            option: config option; one of 'close_figures', 'figure_formats',
+                'print_figure_kwargs', or 'rc'.
+            value: value of the option
         """
+        if (
+            'InlineBackend' in self.config
+            and option in self.config['InlineBackend']
+            and isinstance(value, dict)
+        ):
+            self.config['InlineBackend'][option].update(value)
+        elif 'InlineBackend' in self.config:
+            self.config['InlineBackend'].update({option: value})
+        else:
+            self.config.update({'InlineBackend': {option: value}})
+
+        value = self.config['InlineBackend'][option]
+
+        if isinstance(value, LazyConfigValue):
+            value = value.to_dict().get('update') or value
+
+        self._set_config_option(f'InlineBackend.{option}', value)
+
+        if option == 'rc' and self.get_matplotlib_backend() == 'inline':
+            # Explicitly update rcParams if already in inline mode so that
+            # new settings are effective immediately.
+            import matplotlib
+            matplotlib.rcParams.update(value)
```

Comment:
```suggestion
            try:
                import matplotlib
                matplotlib.rcParams.update(value)
            except Exception:
                pass
```

Just in case Matplotlib fails to import for whatever reason.